common/src/jni/main/cpp/conscrypt/netutil.cc

```diff
@@ -40,44 +34,34 @@
 #endif  // !_WIN32
 
 namespace conscrypt {
+namespace netutil {
 
 /**
- * Network utility methods.
+ * Copied from libnativehelper NetworkUtilites.cpp
  */
-class NetworkUtil {
- public:
-    /**
-     * Copied from libnativehelper NetworkUtilites.cpp
-     */
-    static bool setBlocking(int fd, bool blocking) {
+bool setBlocking(int fd, bool blocking) {
```

Comment:
This is large enough that it probably shouldn't have been inline and so this change makes sense, but, FWIW, there is no requirement that methods in classes be inline or that functions not in classes be out-of-line.

An inline function:
foo.h contains:
```
inline bool setBlocking(int fd, bool blocking) {
  ...
}
```

An out-of-line method:
foo.h contains:
```
class Foo {
 public:
  static bool setBlocking(int fd, bool blocking);
  ...
};
```
in foo.cc:
```
#include "foo.h"

bool Foo::setBlocking(int fd, bool blocking) {
  ...
}
```